Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -417,6 +421,12 @@ int vtkMRMLModelDisplayableManager::UpdateClipSlicesFromMRML()
     this->Internal->YellowSliceClipState = this->Internal->ClipModelsNode->GetYellowSliceClipState();
     }
 
+  if (this->Internal->ClipModelsNode->GetClippingMethod() != this->Internal->ClippingMethod)
+    {
+    modifiedState = 1;
```

Comment:
In this instance, I was following the pattern of the other internal state variables updated in this function that set the modifiedState variable (the return variable of the function).